sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/FormRecognizerRecordedTestSanitizer.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core.Testing;
+
+namespace Azure.AI.FormRecognizer.Tests
+{
+    public class FormRecognizerRecordedTestSanitizer : RecordedTestSanitizer
+    {
+        public override void SanitizeHeaders(IDictionary<string, string[]> headers)
+        {
+            if (headers.ContainsKey(Constants.AuthorizationHeader))
+            {
+                headers[Constants.AuthorizationHeader] = new[] { SanitizeValue };
```

Comment:
Checkout the RecordedTestSanitizer.SanitizeHeaders implementation, it already does the `Authorization` header.

Reply:
The `AuthorizationHeader` here is `Ocp-Apim-Subscription-Key`, which is a different header.